tests/test_bdd.py

```diff
@@ -1,7 +1,17 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 from pytest_bdd import scenario, given, when, then
-# from test_journal import login_helper
+import sys
+
+reload(sys)
+sys.setdefaultencoding('utf-8')
```

Comment:
Does this in fact squash the error?  I would think it would, as it changes the entire behavior of Python :)